src/data/sql-quiz.ts

```diff
@@ -1,4 +1,16 @@
 const sqlQuiz = [
+  {
+    Question: 
+      "How can the user change “codingWebsite” into “FreeCodecamp” in the “Name” column in the Users table?",
```

Comment:
Let's use single quotes here. We also need to use correct capitalization for freeCodeCamp. 

```suggestion
      "In SQL, how can the user change 'codingWebsite' into 'freeCodeCamp' in the 'Name' column for the Users table?",
```